src/containers/profile/Profile.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { connect } from "react-redux";
+import { firestoreConnect } from "react-redux-firebase";
+import { compose } from "redux";
+import { Redirect } from "react-router-dom";
+import { Tabs, Tab, Container, Row, Col } from "react-bootstrap";
+import anonymousImage from "../../images/anonymousImage.png";
+import profileHeaderBG from "../../images/profileHeaderBG.png";
+import Header from "../../components/common/Header";
+import "./Profile.css";
+import AccountSettings from "./ProfileSections/AccountSettings/AccountSettings";
+import ProfilesList from "./ProfileSections/ProfilesList/ProfilesList";
+
+const Profile = (props) => {
+  const { profile, auth, id, isblocked } = props;
+
+  const extraComponents = () => {
+    return (
+      <div className="extraComponents">
+        <h1>
+          {profile.displayName ? profile.displayName : profile.fullName}
+          {profile.jobTitle && (
+            <span className="jobTitle">({profile.jobTitle})</span>
+          )}
+        </h1>
+        {(profile.city || profile.country) && (
+          <div style={{ fontSize: "1.25em" }}>
+            <b>From </b>
+            <span>
+              {profile.city && profile.country
```

Comment:
lines 30-34 are very hard to read, as a guideline lets try to ever avoid using two ternary operators in a row. this should probably be moed to line 16 and just use normal if condition logic